### PR TITLE
[bf/PLAYER-3877] hide volume icon if platform is Ios

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
Hide volume icon if platform is Ios.

Origin task is https://jira.corp.ooyala.com/browse/PLAYER-3877

Unit tests are passed.

### CODE REVIEW

File: tests/components/controlBar-test.js [hunk 1]
```diff
@@ -1112,19 +1112,41 @@ describe('ControlBar', function() {
     expect(slider).not.toBe(null);
   });
 
-  it('hides the volume on iOS', function() {
-    OO_setWindowNavigatorProperty('platform', 'iPhone');
-
-    baseMockProps.skinConfig.buttons.desktopContent = [{'name':'volume', 'location':'controlBar', 'whenDoesNotFit':'keep', 'minWidth':100 }];
+  describe('Hides the volume btn on iOs', function() {
```
Comment (pilievOoyala) [inline tests/components/controlBar-test.js]: Can you change iOs to ios or iOS?

File: tests/components/controlBar-test.js [hunk 2]
```diff
@@ -1112,19 +1112,41 @@ describe('ControlBar', function() {
...
+      expect(typeof wrapper.ref('volumeIcon')).toBe('undefined');
+    });
+
+    it('hides if video is 360', function() {
+      baseMockController.videoVrSource = {};
+      baseMockController.videoVrSource.vr = {};
+
+      var wrapper = Enzyme.mount(getControlBar());
+
+      expect(typeof wrapper.ref('volumeIcon')).toBe('undefined');
```
Comment (pilievOoyala) [inline tests/components/controlBar-test.js]: Just double checking, do we have tests that validate if the volumeIcon shows on non-iOS platforms?

File: tests/components/controlBar-test.js [hunk 3]
```diff
@@ -1112,19 +1112,41 @@ describe('ControlBar', function() {
...
-      <ControlBar {...baseMockProps} controlBarVisible={true}
+    var getControlBar = function(){
+      return (<ControlBar
+        {...baseMockProps}
+        controlBarVisible={true}
         componentWidth={500}
-        playerState={CONSTANTS.STATE.PAUSE}
-      />
-    );
+        playerState={CONSTANTS.STATE.PLAYING}
```
Comment (aeng7) [inline tests/components/controlBar-test.js]: Can we perform the test for both the PLAYING and PAUSED states?

File: tests/components/controlBar-test.js (hunk 2 above)
Comment (KseniiaSokolovaEpam) [reply tests/components/controlBar-test.js]: It was tested in prev test (the line is 1099).

File: tests/components/controlBar-test.js (hunk 1 above)
Comment (KseniiaSokolovaEpam) [reply tests/components/controlBar-test.js]: \+

File: tests/components/controlBar-test.js (hunk 3 above)
Comment (KseniiaSokolovaEpam) [reply tests/components/controlBar-test.js]: \+

**Comment by aeng7** [comment]
LGTM

**Comment by pilievOoyala** [comment]
LGTM.

**Pull request merged**